### ISSUE TITLE
Details about Training Strategy

### ISSUE THREAD

**Issue by BBBoundary** [ali-vilab/MimicBrush#10]
Thanks for your awesome work ! 
As I understand it, you propose Inter-ID and Inner-ID to evaluate the model's imitative editing capabilities between different instances and between the same instances. I noticed that the metrics used by the two benchmarks are different. I wonder how to design the loss function of the model in the model training stage?

**Comment by XavierCHEN34** [comment]
The metrics are different because inner-ID track has ground truth images.  There are only one model trained with traditional diffusion losses, that is the MSE to predict the noise of each step.

**Comment by BBBoundary** [comment]
Thanks，I got it~
